lib/money.js

```diff
@@ -1,15 +1,11 @@
 const CURRENCY_FORMATTERS = {},
-	NUMBER_FORMATTER = new Intl.NumberFormat(undefined, {
-		minimumFractionDigits: 2,
-		maximumFractionDigits: 2
-	}),
-	_getCurrencyFormatter = function (currency) {
+	_getCurrencyFormatter = function (currency, locale) {
 		if (currency == null) {
 			return;
 		}
-		const key = currency.toUpperCase();
+		const key = currency.toUpperCase() + locale;
```

Comment:
If locale is undefined you will get keys like `USDundefined`

```suggestion
		const key = currency.toUpperCase() + (locale || '');
```

Reply:
Followed your suggestion.